app/src/main/java/io/apicurio/registry/rest/v2/AdminResourceImpl.java

```diff
@@ -337,6 +337,9 @@ public RoleMapping getRoleMapping(String principalId) {
     @Authorized(style=AuthorizedStyle.None, level=AuthorizedLevel.Admin)
     @RoleBasedAccessApiOperation
     public void updateRoleMapping(String principalId, UpdateRole data) {
+        requireParameter("principalId", principalId);
+        requireParameter("roleName", data.getRole().name());
```

Comment:
But what if the `role` field is not present or is empty?  We need to check for that.

Also need a unit test I think.

Reply:
Correct, here the NPE is thrown by `data.getRole()` returning `null`.
And I think we need tests 👍 , I hope there is already some infrastructure to mock rbac etc.

Reply:
Thank you for the review @EricWittmann and @andreaTP .
 I'll add null check for Role and add Unit Test as well.